src/ServiceControl.MultiInstance.AcceptanceTests/Recoverability/When_a_message_retry_audit_from_old_sc_is_sent_to_audit_instance.cs

```diff
@@ -0,0 +1,125 @@
+﻿namespace ServiceControl.MultiInstance.AcceptanceTests.Recoverability
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using MessageFailures;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.Pipeline;
+    using NServiceBus.Settings;
+    using NUnit.Framework;
+    using ServiceControl.Infrastructure;
+    using TestSupport;
+    using TestSupport.EndpointTemplates;
+
+    [RunOnAllTransports]
+    class When_a_message_retry_audit_from_old_sc_is_sent_to_audit_instance : AcceptanceTest
```

Comment:
This test removes the `AcknowledgementQueue` header so it should cover two scenarios:
 - Old SC sending a retry message to a new endpoint
 - New SC sending a retry message to an old endpoint
 
 In both of these cases the endpoint logic is not executed and we end up going through the audit path.

Reply:
discussed this with Szymon on the sync. We think these test cases could actually be done as audit acceptance tests instead of multi-instance tests, as we don't really need the full setup to test the audit behavior. The only need for a multi-instance test is to verify that an enabled audit instance does not mess up with the new retry confirmation path (which only requires the main instance to work), e.g. by messing up query data (through duplicates).